pipelines/optimus/Optimus.wdl

```diff
@@ -1,140 +1,151 @@
-import "FastqToUBam.wdl" as fq2bam
-import "Attach10xBarcodes.wdl" as attach
-import "SplitBamByCellBarcode.wdl" as split
-import "MergeSortBam.wdl" as merge
-import "CreateCountMatrix.wdl" as count
-import "StarAlignBamSingleEnd.wdl" as StarAlignBamWDL
-import "TagGeneExon.wdl" as TagGeneExonWDL
-import "CorrectUmiMarkDuplicates.wdl" as CorrectUmiMarkDuplicatesWDL
-import "SequenceDataWithMoleculeTagMetrics.wdl" as metrics
+import "FastqToUBam.wdl" as FastqToUBam
+import "Attach10xBarcodes.wdl" as Attach
```

Comment:
What are your thoughts on giving this a more descriptive name, such as `AttachBarcodes` or `Attach10xBarcodes`? Same with the ones below like `Split`, `Merge` and `Count`

Reply:
Actually looking at the rest of the file I think the descriptive task names are helpful enough!